lms/djangoapps/certificates/views/webview.py

```diff
@@ -248,6 +250,11 @@ def _update_course_context(request, context, course, platform_name):
                                                               '{partner_short_name}.').format(
             partner_short_name=context['organization_short_name'],
             platform_name=platform_name)
+    # If language specific templates are enabled for the course, add course_run specific information to the context
+    if CertificateGenerationCourseSetting.is_language_specific_templates_enabled_for_course(course_key):
```

Comment:
Is there ever a case where we wouldn't have language-specific templates enabled but we'd want to show hours of effort?